pkg/pillar/cmd/nim/nim.go

```diff
@@ -1057,6 +1057,9 @@ func (n *nim) includeLastResortPort(ifAttrs netmonitor.IfAttrs) bool {
 	exclude := strings.HasPrefix(ifName, "vif") ||
 		strings.HasPrefix(ifName, "nbu") ||
 		strings.HasPrefix(ifName, "nbo") ||
+		strings.HasPrefix(ifName, "can") ||
```

Comment:
For CAN maybe you can check if `ifAttrs.IfType` is `can` instead of relying on interface name? I cannot test it, but it seems that there is "can" interface type, see "Link types" in [iplink](https://man7.org/linux/man-pages/man8/ip-link.8.html))

Reply:
Yeah, you are right, I will update the PR to use `ifAttrs.IfType`....